content/posts/cneuromonth_videos.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "CNeuromonth videos"
```

Comment:
I'll stop being annoying but I wonder if the title can be something like CNeuromonth 2021 - the videos!
I'm just thinking of how it is going to look on the blog main page..
![Screen Shot 2021-04-22 at 8 20 22 AM](https://user-images.githubusercontent.com/11749857/115713128-a74e6080-a343-11eb-9be4-951a99851538.png)

Reply:
provide feedback = awesome $/neq$ annoying :hugs: :hugs: :hugs: 

I agree with proposed title.